### ISSUE TITLE
[REQUEST] Get playtime from Steam

### ISSUE THREAD

**Issue by JustinAiken** [sandiz/rs-manager#24]
Rocksmith doesn't count score attack/menus/etc towards playtime.. it's off by about 400 hours for me!

You can get playtime in minutes from steam using [node-steam-webapi](https://github.com/jonbo/node-steam-webapi) like:

```es6
const Steam = require('steam-webapi');

Steam.key = "much_secret"; 

const getPlaytime = (steamID) => {
  Steam.ready(() => {
    const steam = new Steam();

    steam.getOwnedGames({
      appids_filter: [221680],
      steamid: steamID,
      include_played_free_games: false,
      include_appinfo: 1,
    }, (err, data) => {
      const minutesPlayed = data.games[0].playtime_forever;
      console.log(minutesPlayed);
    });
  });
};

getPlaytime(steam_user_id);
```

The downside is it requires an API key - so maybe make an optional setting, and fallback to Rocksmith's stat if not entered.. 
I'm not sure if you can get playtime via the cookie method you use to get owned packages...

Bonus points if you can config playtime display for Days/Hours/Minutes or just Hours/Minutes!

**Comment by sandiz** [comment]
fixed in [`cea5b28`](https://github.com/sandiz/rs-manager/commit/cea5b28c8e3b53b529602f67dd2c4df634b90f31), will be in the next release

**Comment by JustinAiken** [comment]
In order to make it work, had to put my `STEAM_API_KEY` in the `internal.json` *before* building the release... but then it works perfect!

![steamtime](https://user-images.githubusercontent.com/1568662/47606815-940de480-d9d5-11e8-8a32-8d52669da434.png)

**Comment by sandiz** [comment]
👍 i'll put the key in the when i release (wont checkin tho)

**Comment by sandiz** [comment]
agreed, i'll keep the api key out of the build and provide some documentation instead.

i was kinda half a mind about it anyway, on one hand its a bad security practice, on the other hand its really easy to get a key from steam website, so why bother reversing

**Comment by sandiz** [comment]
fixed in release [v1.1.0](https://github.com/sandiz/rs-manager/releases/tag/v1.1.0)

**Comment by JustinAiken** [comment]
> agreed, i'll keep the api key out of the build

If keeping it out of the build, may want to move it from the `internal.json` to a regular user setting, so people can paste in their key without having to do the electrony dance
